osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -660,9 +621,22 @@ - (void)showPreferences {
 }
 
 
-//- (void)loadPreferences {
-    // TODO(amodia): Set custom database path in the preferences menu.
-//}
+- (void)loadPreferences {
+    
+    NSString *customKaliteData = getCustomKaliteHomePath();
+    
+    if (pathExists(customKaliteData)) {
+        NSString *standardizedPath = [customKaliteData stringByStandardizingPath];
+        self.customKaliteData.stringValue = standardizedPath;
+    } else {
+        NSString* envKaliteHomeStr = getEnvVar(@"KALITE_HOME");
+        if (pathExists(envKaliteHomeStr)) {
+            self.customKaliteData.stringValue = envKaliteHomeStr;
+        } else {
+            self.customKaliteData.stringValue = [NSString stringWithFormat:@"%@/.kalite", NSHomeDirectory()];
```

Comment:
I think these codes must be the behaviour of the `getCustomKaliteHomePath()` function.  And if it returns an empty string, you must notify the user to "Start KA Lite" because it will auto-create the `~/.kalite/` data folder.